nodes/celestia-node-troubleshooting.md

```diff
@@ -78,6 +78,33 @@ If you would like to use them anyway, you can
 
 ## Changing the location of your node store
 
+### Background
+
+An [enhancement has been made in v0.14.0+](https://github.com/celestiaorg/celestia-node/pull/3246)
+to automate the detection of the running node, eliminating the need to
+manually specify the `--node.store` flag for each RPC request.
+
+**Assumptions:**
+
+- The presence of a lock signifies a running node.
```

Comment:
I don't think you need to mention this assumption: - presence of lock (implementation detail).